src/LanguageServer.TaskReflection/Program.cs

```diff
@@ -101,98 +101,96 @@ public static int Main(string[] args)
                     )
                     .ToArray();
 
-                using (StringWriter output = new StringWriter())
```

Comment:
Diff is always drunk when it comes to massive ident changes (due to simplifying `using` refactoring)